fluent/ExtrinsicRune.ts

```diff
@@ -46,20 +49,35 @@ export class ExtrinsicRune<out C extends Chain, out U> extends PatternRune<Chain
       .call(this.chain.into(ValueRune).access("metadata"))
       .into(CodecRune)
       .encoded(Rune.rec({
-        protocolVersion: 4,
+        protocolVersion: ExtrinsicRune.PROTOCOL_VERSION,
         call: this,
       }))
   }
 
   feeEstimate() {
-    const extrinsicHex = this.encoded().map(hex.encodePrefixed)
-    return this.chain.connection.call("payment_queryInfo", extrinsicHex)
-      .map(({ weight, ...rest }) => ({
-        ...rest,
-        weight: {
-          proofSize: BigInt(typeof weight === "number" ? 0 : weight.proof_size),
-          refTime: BigInt(typeof weight === "number" ? weight : weight.ref_time),
-        },
-      }))
+    const $queryInfoCallArgs = Rune.fn($extrinsic)
+      .call(this.chain.into(ValueRune).access("metadata"))
+      .map((ext) => $.tuple(ext, $.u32))
```

Comment:
I wonder if we could/should tweak [SCALE's `lenPrefixed` codec](https://github.com/paritytech/scale-ts/blob/dc64f6564044b8626535d717c9c2bc8e24cc6a9a/codecs/lenPrefixed.ts) to support postfixing instead / specificity of direction?